src/main/java/models/AuditEvent.java

```diff
@@ -0,0 +1,18 @@
+package models;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+@Getter
+@Setter
+@ToString
+public class AuditEvent {
+    public String auditEventId;
+    public String source;
+    public String timestamp;
+    public String type;
+    public String workspaceId;
```

Comment:
Why are we exposing `workspaceId`? This is not a concept for our customers

Reply:
Fixed!